server/src/createContext.ts

```diff
@@ -36,9 +38,15 @@ export const createOffixMongoCRUDRuntimeContext = (
         }
 
         const objectDB = new OffixMongoDBDataProvider(modelType, db)
-        services[model.name] = new AMQCRUDService(modelType, objectDB, {
-            pubSub,
-            ...model.pubSub
+
+        services[model.name] = new AMQCRUDService({
+            modelType,
+            db: objectDB,
+            subscriptionConfig: {
+                pubSub,
+                ...model.pubSub
+            },
+            authConfig: authConfig[model.name]
```

Comment:
@wtrocki trying to figure out how to document the usage of the library.

This stuff is happening inside a helper function that we copy pasted into the application. Normally the user doesn't really write the code to create all those services.

I can write a helper function in the original library that looks like this one but accepts a DataProvider constructor. That way the helper can create everything fairly easily. All of the existing DataProvider classes in the Graphback Project currently take the same args but it's not actually defined which means it's not really guaranteed. Any ideas?

The API here for setting up the runtime is getting quite convoluted. I had a PR to try address it but it was closed.

Reply:
Please connect this any way without spending much time. 
Helper works great for me at the moment.
We will refine it once this lands.